lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -1341,3 +1388,316 @@ def _capture_libvirt_logs(self) -> None:
             )
             with open(str(libvirt_log_local_path), "w") as f:
                 f.write(libvirt_log)
+
+
+class LibvirtDevicePoolImpl(BaseDevicePoolImpl):
+    def __init__(
+        self,
+        node: Union[Node, RemoteNode],
```

Comment:
`RemoteNode` inherits from `Node`. So, you don't need the `Union` here. You can just say `Node`.

Reply:
Corrected